src/unit_tests/wazuh_db/test_wdb_upgrade.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2015-2020, Wazuh Inc.
+ * September, 2020.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "wazuh_db/wdb.h"
+#include "../wrappers/common.h"
+#include "../wrappers/wazuh/shared/debug_op_wrappers.h"
+#include "../wrappers/wazuh/wazuh_db/wdb_metadata_wrappers.h"
+#include "../wrappers/wazuh/wazuh_db/wdb_wrappers.h"
+
+typedef struct test_struct {
+    wdb_t *socket;
+    char *output;
+} test_struct_t;
+
+/* setup/teardown */
+
+int setup_wdb(void **state) {
+    test_struct_t *init_data = NULL;
+    os_calloc(1,sizeof(test_struct_t),init_data);
+    os_calloc(1,sizeof(wdb_t),init_data->socket);
+    os_strdup("000",init_data->socket->id);
+    os_calloc(256,sizeof(char),init_data->output);
+    os_calloc(1,sizeof(sqlite3 *),init_data->socket->db);
+    *state = init_data;
+    return 0;
+}
+
+int teardown_wdb(void **state) {
+    test_struct_t *data  = (test_struct_t *)*state;
+    os_free(data->output);
+    os_free(data->socket->id);
+    os_free(data->socket->db);
+    os_free(data->socket);
+    os_free(data);
+    return 0;
+}
+
+/* Tests wdb_upgrade_global */
+
+void test_wdb_upgrade_global_table_fail(void **state)
+{   
+    wdb_t *ret = NULL;
+    test_struct_t *data  = (test_struct_t *)*state;
+    
+    expect_string(__wrap_wdb_metadata_table_check, key, "metadata");
+    will_return(__wrap_wdb_metadata_table_check, OS_INVALID);
+    expect_string(__wrap__mdebug1, formatted_msg, "DB(000) Failed trying to find metadata table");
```

Comment:
Here the right expression should be either `Failed to try to...` or `Error trying to...` or `Failure trying to...`

Reply:
Fixed. Thank you!